polaris/management/commands/testnet.py

```diff
@@ -320,3 +417,162 @@ def get_balance(self, code, issuer_public_key, json) -> Optional[str]:
                 and balance_obj.get("asset_issuer") == issuer_public_key
             ):
                 return balance_obj["balance"]
+
+    def create_trustline(self, code :str, asset_issuer_public_key: str, distribution: Keypair):
```

Comment:
```suggestion
    def create_trustline(self, code: str, asset_issuer_public_key: str, distribution: Keypair):
```

It may be helpful to run `black` on this code to format it